README.md

```diff
@@ -721,23 +721,31 @@ crud.stats()
     my_space:
       insert:
         ok:
-          latency: 0.002
+          latency: 0.0015
+          latency_average: 0.002
+          latency_quantile_recent: 0.0015
```

Comment:
I'll not insist here, but, as for me, `latency_recent` is descriptive enough.

Reply:
I think it's better to leave it since "quantile" is what this value is.